### ISSUE TITLE
[Timers] 'Segmentation fault' output on Linux

### ISSUE THREAD

**Issue by xiaolongx** [01org/zephyr.js#682]
'Segmentation fault' output and then exit automatically when running [Timers.js](https://github.com/01org/zephyr.js/blob/master/samples/Timers.js) via command: `./outdir/linux/release/jslinux samples/Timers.js` with commit 21915c0, this issue cannot reproduce on both Arduino 101 and FRDM-K64F.

```
Starting Timers example...
Interval (args) #1 arg1: 1 arg2: 2
Interval (no args) #2
Interval (args) #2 arg1: 1 arg2: 2
Interval (no args) #3
Interval (args) #3 arg1: 1 arg2: 2
Interval (no args) #4
Interval (args) #4 arg1: 1 arg2: 2
Interval (no args) #5
Timeout, clearing interval
Segmentation fault (core dumped)

```

**Comment by qiaojingx** [comment]
This issue is not reproduced Linux with commit 3f8eb85.

**Comment by xiaolongx** [comment]
This issue is reproduced on Linux again with commit 7c5ff3c.
Reopen.

**Comment by grgustaf** [comment]
Should be fixed by #780.

Also, the output of this test will change with #781. Note the #1/#2 output is strange: 1-2-2-3-3-4-5. It will be 1-1-2-2-3-3-4-4-5-5 now, which is what happens on Chrome and probably what the author intended.

**Comment by qiaojingx** [comment]
Verified with commit b29dbbc, this issue is not reproducible when running Timers.js on Linux.
But 'Segmentation fault' issue occurred when running test-timers.js and ocfServer.js on Linux.
Create a new issue #792.
